snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -135,6 +152,12 @@ open class Snapshot: NSObject {
         #if os(OSX)
             XCUIApplication().typeKey(XCUIKeyboardKeySecondaryFn, modifierFlags: [])
         #else
+            
+            guard let app = self.app else {
+                print("XCUIApplication is not set")
```

Comment:
This error message could be a little clearer so that the user knows what they have to do to fix the error 👍